pkg/strategy/xgap/strategy.go

```diff
@@ -260,56 +271,82 @@ func (s *Strategy) makeSpread(ctx context.Context, bestBid, bestAsk types.PriceV
 	return err
 }
 
-func getAdjustPositionOrder(symbol string, tradingMarket types.Market, currentPosition *types.Position, bestBid, bestAsk types.PriceVolume) (orderForms []types.SubmitOrder) {
-	basePosition, avgCost := currentPosition.GetBaseAndAverageCost()
-	if basePosition.Eq(fixedpoint.Zero) {
+// getAdjustPositionOrders returns the order forms that can adjust the current position if possible.
+// When in a short position, it will try to place a buy order.
+// When in a long position, it will try to place a sell order.
+// It's based on the spread and the condition of the current position.
+func getAdjustPositionOrders(
```

Comment:
you are generating an new order, not returning an existing order, so using prefix "get" could be confusing

you may use "build" or "new" as the prefix